### ISSUE TITLE
Resolvers: Create Tests for ReuploadDuplicateCheck.js

### ISSUE THREAD

**Issue by SiddheshKukade** [PalisadoesFoundation/talawa-api#795]
The Talawa-API code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/helper_functions/ReuploadDuplicateCheck.js`

- We will need the API to be refactored for all methods, classes and/or functions found in this file for testing to be correctly executed.
- When complete, all all methods, classes and/or functions in the refactored file will need to be tested. These tests must be placed in a
single file with the name `tests//helper_functions/ReuploadDuplicateCheck.js`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa-api/issues/490

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa-api/tree/develop/lib/resolvers/organization_query/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.

**Comment by TheLazron** [comment]
Can I be assigned this issue, would like to work on it

**Comment by kb-0311** [comment]
pls assign it to me

**Comment by palisadoes** [comment]
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by kb-0311** [comment]
@palisadoes Sir , what do you mean by 'coordinate issue assignment ' with them? Like ping them personally or tag them in the issue assignment comment under an issue?

**Comment by palisadoes** [comment]
Yes. They have volunteered to assist me with this. The workload was getting too great for me alone.

**Comment by noman2002** [comment]
@TheLazron are you still working on this ??

**Comment by SiddheshKukade** [comment]
@noman2002  It's been 2 weeks since @TheLazron is working on this. I would like to take this up. you can assign to me

**Comment by kb-0311** [comment]
@SiddheshKukade This is a duplicate Issue , same as #887 for which I was assigned.

I have already made a PR for this file https://github.com/PalisadoesFoundation/talawa-api/pull/963
